docs/releases/upgrading/upgrade-v12.0.md

```diff
@@ -0,0 +1,215 @@
+# Upgrading from v11 to v12
+
+## Prerequisites
+
+The steps listed in this article require an existing local installation of InvenioRDM v11.
+
+!!! warning "Backup"
+
+    Always backup your database and files before you try to perform an upgrade.
+
+!!! info "Older Versions"
+
+    In case you have an InvenioRDM installation older than v11, you can gradually upgrade
+    using the existing infrastructure to v11 and afterwards continue from here.
+
+## Upgrade Steps
+
+!!! warning "Upgrade your invenio-cli"
+
+    Make sure you have the latest `invenio-cli` installed. For InvenioRDM v12 it
+    is at least v1.2.0
```

Comment:
Any reason not to tell folks to go to 1.3.1?

Reply:
i think at the point i wrote that, v1.2.0 was the newest. i will change that!